chrome_extension/cardboard/popup.js

```diff
@@ -1,10 +1,37 @@
+const IS_PROD = true;
+
+const PROD_URL = "https://cardboard.rocks";
+const DEV_URL = "http://localhost:8000";
+
+// TODO: Make this configurable
+const HUNT_NUMBER = 10;
+
+let URL = DEV_URL;
```

Comment:
`URL` already exists as a global, we shouldn't shadow it https://developer.mozilla.org/en-US/docs/Web/API/URL/URL

Reply:
Changed to TARGET_URL